docs/dom-testing-library/api-queries.md

```diff
@@ -581,6 +586,24 @@ Library uses `aria-query` under the hood to find those elements by their default
 ARIA roles, you can find in their docs
 [which HTML Elements with inherent roles are mapped to each role](https://github.com/A11yance/aria-query#elements-to-roles).
 
+You can query the returned element(s) by their
+[accessible name](https://www.w3.org/TR/accname-1.1/). The accessible name is
+for simple cases equal to e.g. the label of a form element, or the text content
+of a button, or the value of the `aria-label` attribute. It can be used to query
+a specific element if multiple elements with the same role are present on the
+rendered content. For an in-depth guide check out
+["What is an accessible name?" from ThePacielloGroup](https://developer.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/).
+If you only query for a single element with `getByText('The name')` it's
+oftentimes better to use `byRole(expectedRole, { name: 'The name' })`. The
+accessible name query does not replace other queries such as `byAlt` or
+`byTitle`. While the accessible name can be equal to these attributes, it does
```

Comment:
Let's make sure that we never have sample code that looks like it could be real code but isn't. So either `getByRole...` or `*ByRole...`

Reply:
Done